plasmapy/physics/transport.py

```diff
@@ -12,17 +12,64 @@
 from plasmapy.atomic.atomic import _is_electron
 from .parameters import (Debye_length, Hall_parameter,
                          collision_rate_electron_ion, collision_rate_ion_ion)
+from .quantum import (Wigner_Seitz_radius,
+                      thermal_deBroglie_wavelength,
+                      chemical_potential)
+from ..mathematics import Fermi_integral
 from inspect import stack
 from copy import copy
 import warnings
 
 
-@utils.check_quantity({"T": {"units": units.K, "can_be_negative": False},
-                       "n_e": {"units": units.m**-3}
-                       })
-def Coulomb_logarithm(T, n_e, particles, V=None):
-    r"""
-    Estimates the Coulomb logarithm.
+
+
+def _boilerPlate(T, particles, V):
```

Comment:
This would be a good candidate for refactoring, as it  does three visibly different things:
* temperature check
* particle check that would probably be alleviated at least a bit by #239 
* reduced mass (perhaps that could be incorporated into the `Particle` API, @namurphy )?
But that refactoring is no priority here and could be done in a later PR.

Reply:
Oh...that's a good idea to have a reduced mass thing in the Particle class since that comes up a lot.

Reply:
Is splitting up the temperature and particle checks really necessary? I mean all of these things get called in each of my transport functions so it's much easier to call a single function with all the boiler plate.

How would you envision refactoring this one? Multiple individual check functions that each need to be called in each of the transport functions or just refactoring within `_boilerplate` to incorporate more of the particle API stuff?

Reply:
I guess I'm fine with keeping the `_boilerplate` function for now, though I think it's sort of suboptimal in a way, in that it makes the code less readable for the future... I'm not completely sure. If we had ways to quickly do all the stuff that `_boilerplate` currently does, even in four lines of code if need be... That might be reasonable, no?

I also managed to completely miss the thermal velocity in this function, so that's where "four lines of code" comes from. It has been pretty cold lately, so that's probably why I neglected it.